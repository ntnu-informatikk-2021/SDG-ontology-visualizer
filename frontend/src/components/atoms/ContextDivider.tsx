import { Center, Divider } from '@chakra-ui/react';
import React from 'react';

type ContextDividerProps = {
  visible: boolean;
};

const ContextDivider: React.FC<ContextDividerProps> = ({ visible }: ContextDividerProps) => {
  if (!visible) return <></>;
  return (
    <Center mx={[10, null, null, 20]}>
      <Divider orientation="vertical" />
    </Center>
  );
};

export default ContextDivider;
