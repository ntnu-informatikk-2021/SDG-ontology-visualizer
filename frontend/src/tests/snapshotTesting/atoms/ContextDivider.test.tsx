import React from 'react';
import renderer from 'react-test-renderer';
import ContextDivider from '../../../components/atoms/ContextDivider';

it('renders when there are no items', () => {
  const tree = renderer.create(<ContextDivider visible />).toJSON();
  expect(tree).toMatchSnapshot();
});
