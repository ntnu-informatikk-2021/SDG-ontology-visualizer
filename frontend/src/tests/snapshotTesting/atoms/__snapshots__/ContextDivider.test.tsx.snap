// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there are no items 1`] = `
<div
  className="css-1fo6ot5"
>
  <hr
    aria-orientation="vertical"
    className="chakra-divider css-12gj880"
    role="separator"
  />
</div>
`;
